### DISCUSSION TITLE
.NET MAUI Development in Codespaces

### DISCUSSION THREAD

**Discussion by rogersm-uwosh** [dotnet/maui#23996] [Q&A]
Is there anyway to do .NET MAUI development in a Codespace? A huge problem I have is that my students and I are not always using the exact same environment, and I have to futz to get their projects to work. Codespaces looks like it might (?) work on Android, and it would be fantastic for collaboration.

**Comment by jingliancui** [comment]
I think maybe the tricky approach is make the compiling work in the clouds pipeline
